ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/LagReportingAgent.java

```diff
@@ -174,9 +172,9 @@ public void receiveHostLag(final LagReportingMessage lagReportingMessage) {
       final HostInfo host, final QueryStateStoreId queryStateStoreId, final int partition) {
     final Set<HostInfo> aliveHosts = aliveHostsRef.get();
     final LagInfoEntity lagInfo = receivedLagInfo
-        .getOrDefault(host, EMPTY_HOST_LAG_INFO).getLagInfo()
-        .getOrDefault(queryStateStoreId, Collections.emptyMap())
-        .getOrDefault(partition, null);
+        .getOrDefault(host, EMPTY_HOST_STORE_LAGS)
+        .getStateStoreLagsOrDefault(queryStateStoreId, EMPTY_STATE_STORE_LAGS)
```

Comment:
why not more `EMPTY_STATE_STORE_LAGS` into `getStateStoreLags`?

Reply:
I'll change it to return an optional, as suggested below.